static/app/components/searchQueryBuilder/tokens/filter/valueCombobox.tsx

```diff
@@ -93,7 +94,7 @@ const DEFAULT_NUMERIC_SUGGESTIONS: SuggestionSection[] = [
 const DEFAULT_DURATION_SUGGESTIONS: SuggestionSection[] = [
   {
     sectionText: '',
-    suggestions: [{value: '100'}, {value: '100k'}, {value: '100m'}, {value: '100b'}],
+    suggestions: DURATION_UNITS.map(unit => ({value: `100${unit}`})),
```

Comment:
These defaults feel weird for the days/weeks - 100d/100w won't really ever find anything given our 90d retention. Aside from special casing all of them the only suggestion I have would be to maybe change to `10${unit}`

Reply:
That's true, I had similar thoughts. I doubt days or weeks will every be useful for the filters we have, but it also seems like we should show them as an option somewhere.

I think I'll change to 10 as you suggested, and perhaps remove d and w from the default suggestions